### ISSUE TITLE
disable keybinding for ctrl-f

### ISSUE THREAD

**Issue by mokraemer** [phpmyadmin/phpmyadmin#13798]
### Steps to reproduce
1. have a big table with many columns
2. use select * from table where xx
3. press ctrl-f to use browser search to have a quick highlight of specific coulumns / values

### Expected behaviour
open browser search (window) field, no change of search result

### Actual behaviour
browser field is shown, Search dialog is is shown, result table is discarded

**phpMyAdmin version:**
4.7.1
### Client configuration
Opera

**Operating system:**
Linux

**Comment by nijel** [comment]
I can't reproduce this on Firefox, but probably result of keyboard shortcuts:

https://github.com/phpmyadmin/phpmyadmin/blob/d3f9075165236b363c188e8d4ce1adf5a1e6c7b7/js/shortcuts_handler.js#L40-L100

The question is how to have keyboard shortcuts and avoid collision with browser ones...

**Comment by mokraemer** [comment]
You're right, in firefox the shortcut is blocked by the browser's own.
I don't think it is a good idea to have additional shortcuts for functions that can be reached by one click. Or at least, please have a config option to disable such features. It might have been a good idea, but currently it is more a barrier, than a help (since my previous query is discarded).

**Comment by Jamibaraki** [comment]
I've written some code to add the config option to disable keyboard shortcuts altogether. Is this the approach people want to take, or is it better to change the shortcuts to avoid the clashes?

**Comment by flohw** [comment]
Hi,

I just found this issue as firefox seems to not block the shortcut anymore: we are redirected to search tab and the search box is shown at the bottom.

Could be very nice if the shortcuts could be disabled through PMA or customizable.

Thanks.

**Comment by williamdes** [comment]
@MauricioFauth would you agree for another config file setting to disable all the shortcut JS listeners?

**Comment by mokraemer** [comment]
I guess it is the best option, as the browser catches almost all possible short-cuts and not all browsers use the same

**Comment by MauricioFauth** [comment]
> @MauricioFauth would you agree for another config file setting to disable all the shortcut JS listeners?

I agree. But the config should be to enable shortcuts and be enabled by default instead of a setting to disable the shortcuts.

**Comment by williamdes** [comment]
I did not know but there is already a config setting to disable all key bindings: https://docs.phpmyadmin.net/en/latest/config.html#cfg_DisableShortcutKeys

**Comment by mokraemer** [comment]
nor did I. and maybe this is because the setting is not shown in 
https://github.com/phpmyadmin/phpmyadmin/blob/master/config.sample.inc.php
which is usally the point where to look for parameters.

**Comment by mokraemer** [comment]
maybe this file should be updated, to reflect new possible settings?

**Comment by williamdes** [comment]
> maybe this file should be updated, to reflect new possible settings?

I am not sure what is the policy to add "most used" settings to this file

**Comment by williamdes** [comment]
I have a patch locally, to ensure CTRL or ALT are not used when `f` is used. Maybe it should fix #17291 
Can you still reproduce this issue you did mention ?
If yes please provide full details as it will help me a lot
